src/bitmessagekivy/mpybit.py

```diff
@@ -319,6 +319,16 @@ def reset_login_screen(self):
         if self.root.ids.id_newidentity.ids.add_random_bx.children:
             self.root.ids.id_newidentity.ids.add_random_bx.clear_widgets()
 
+    def set_common_header(self):
+        """Common header for all windows"""
```

Comment:
better docstring, this is too general/inaccurate

Reply:
Now this a common header which have two icons one is for side nav bar and one for add new contact address so this will appear in all screens if any address is already created